homeassistant/components/niko_home_control/light.py

```diff
@@ -51,6 +56,9 @@ async def async_setup_platform(
 class NikoHomeControlLight(LightEntity):
     """Representation of an Niko Light."""
 
+    _attr_color_mode = ColorMode.ONOFF
+    _attr_supported_color_modes = {ColorMode.ONOFF}
```

Comment:
Should it not be `ColorMode.BRIGHTNESS`?

Reply:
Setting it to ONOFF will not change the behavior.

The non-working brightness code has been there since the integration was first added, @MartinHjelmare had review comments about it which were never addressed: <https://github.com/home-assistant/core/pull/18019>

Maybe @NoUseFreak or @legovaer could comment?